Cargo.toml

```diff
@@ -7,10 +7,8 @@ build = "build.rs"
 
 [dependencies]
 libc = "*"
-rust-htslib = { version = "0.26", default-features = false, features = ["serde"] }
```

Comment:
This will cause us to link against a different version of rust-htslib, AFAIU--does this cause problems with passing e.g. bam::Record's around?

Reply:
I believe that features are ignored for purposes of resolving dependencies, then the crate is built with the union of the features requested by all users.  So I don't think this causes us to get a different `rust-htslib` than the rest of the build.